server/static/bindata_assetfs.go

```diff
@@ -265,7 +265,7 @@ var _bindata = map[string]func() (*asset, error){
 //         b.png
 // then AssetDir("data") would return []string{"foo.txt", "img"}
 // AssetDir("data/img") would return []string{"a.png", "b.png"}
-// AssetDir("foo.txt") and AssetDir("notexist") would return an error
+// AssetDir("foo.txt") and AssetDir("nonexistent") would return an error
```

Comment:
this is still being modified

Reply:
I made the fix on master

Reply:
Oops, sorry. I should have dropped the file -- I'm usually better at that.